digits/config/caffe_option.py

```diff
@@ -242,6 +242,9 @@ def apply(self):
                 os.environ['PYTHONPATH'] = '%s:%s' % (p, os.environ.get('PYTHONPATH'))
 
             try:
+                # for Windows environment, loading h5py before caffe solves the issue mentioned in
+                # https://github.com/NVIDIA/DIGITS/issues/47#issuecomment-206292824
+                import h5py
```

Comment:
Can you put this before the try/catch block? If someone didn't install h5py, then they would get a misleading error: `Did you forget to "make pycaffe"?`.